server/dashboard.js

```diff
@@ -443,7 +443,7 @@ async function submitNewClient(req, res) {
         true,
         false,
         false,
-        'en',
+        req.body.language || 'en',
```

Comment:
yes, the expression ` req.body.language || 'en'` will resolve to `'en'` when `req.body.language` is `undefined or has any falsy value such as `null` or empty string. if the value of `req.body.language` is defined and truthy, it will resolve to its value as in the case when selecting Spanish language.